omnigibson/prims/entity_prim.py

```diff
@@ -169,24 +160,24 @@ def _post_load(self):
 
         self._materials = materials
 
-    def update_links(self, load_config=None):
+    def update_links(self):
         """
         Helper function to refresh owned joints. Useful for synchronizing internal data if
         additional bodies are added manually
-
-        Args:
-            load_config (None or dict): If specified, should contain keyword-mapped values that are relevant for
-            loading each of the link's rigid prims
         """
+        load_config = {
```

Comment:
I think the original idea was to make this method more functional in nature, but you're right that this is better in the sense that if a user calls it externally, it's less error-prone